icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -22,11 +20,31 @@
 import com.ibm.icu.util.ULocale;
 import com.ibm.icu.util.UResourceBundle;
 
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.MissingResourceException;
+
 public class LongNameHandler implements MicroPropsGenerator, ModifierStore {
 
-    private static final int DNAM_INDEX = StandardPlural.COUNT;
-    private static final int PER_INDEX = StandardPlural.COUNT + 1;
-    private static final int ARRAY_LENGTH = StandardPlural.COUNT + 2;
+    protected static final int DNAM_INDEX = StandardPlural.COUNT;
+    protected static final int PER_INDEX = StandardPlural.COUNT + 1;
+    protected static final int ARRAY_LENGTH = StandardPlural.COUNT + 2;
+    private final Map<StandardPlural, SimpleModifier> modifiers;
+    private final PluralRules rules;
+
+
+    private final MicroPropsGenerator parent;
+
+    // NOTE: outArray MUST have at least ARRAY_LENGTH entries. No bounds checking is performed.
```

Comment:
Which outArray where? This comment is now on line 38, the first reference to outArray is on line 319.

This comment used to be between PluralTableSink and getMeasureData. Maybe not the best of places, but here it's totally pointless. Please review your own changes and try to catch these things before you ask others to catch them for you?

I'm moving it back into the place it was.

Reply:
Moved in 08744e0867